src/overlays/actors/ovl_En_Test6/z_en_test6.c

```diff
@@ -15,14 +16,31 @@ void EnTest6_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnTest6_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnTest6_Draw(Actor* thisx, GlobalContext* globalCtx);
 
+struct EnTest6Struct;
```

Comment:
Can't do that because EnTest6Struct references EnTest6StructFunc, which takes a pointer to EnTest6Struct. So I would still need a forward. Unless I delete the function typedef and inline that everywhere, but I think we usually typedef them? I dunno, what do you think?

Reply:
What about putting `EnTest6StructFunc`  and the struct definition up here together?